### ISSUE TITLE
Expanding Editor to multiple files

### ISSUE THREAD

**Issue by TPreece101** [streamsync-cloud/streamsync#77]
I was wondering what your thoughts are on expanding the editor to multiple files, It seems like it's currently intended for quick edits and so this might not be something you're interested in.

However, being able to do all of your required edits in the Builder would make a very complete app building experience. I've never been a fan of hopping around between windows a lot when trying to build something.

**Comment by ramedina86** [comment]
It's certainly in the roadmap to improve the editor, but no short-term plans for this. Adding the file switcher is just part of the equation; I'd feel compelled to also support autocomplete and various functionality before calling it a "real editor".

**Comment by TPreece101** [comment]
Yeah that makes sense, although autocomplete does sound like a much larger task 😬
